docs/tutorial/hyperfoil.adoc

```diff
@@ -1,129 +1,156 @@
 = Scripting hyperfoil with quarkus getting started
 
-The `curl` command from link:quarkusgetstarted.adoc[Scripting Quarkus getting started] was a substitute for a load driver. We are going to use hyperfoil to drive load to our quarkus application and use qDup `setup-scripts` and `cleanup-scripts` phases to coordinate the test.
+The `curl` command from link:quarkusgetstarted.adoc[Scripting Quarkus getting started] was a substitute for the load generator. We are going to use link:https://hyperfoil.io[Hyperfoil] to drive the load to our quarkus application and use qDup phases to coordinate the test.
 
 == Introduction
 
-This tutorial will introduce us to:
+This tutorial will introduce:
 
-1. Commands that change the ssh shell prompt
-2. qDup `setup-scripts` and `cleanup-scripts`.
-3. qDup `roles`
+1. link:./../reference/command/addprompt.adoc[qDup add-prompt]
+2. link:./../reference/roles.adoc[qDup roles]
 
+== Prerequisites
 
-== Where we left off
+qDup connects to computers through ssh. You need to have a computer that will accept ssh connections. You can setup ssh on your own computer by following the steps in the link:./prerequisites.adoc[prerequisites]
 
-The previous tutorial created a qDup yaml that downloaded Quarkus getting started and ran a single curl against the `quarkus dev` process. We are going to start where it left off:
-
-```yaml
+== Running the experiment
```

Comment:
Please add a sentence after the heading as a code section immediately after the heading can leave a user to scroll past the code section to find out why the code section exists then scrolling back up to the code section to interact with it.

Reply:
Done